tests/fixtures/cp/protected.txt

```diff
@@ -0,0 +1 @@
+new text
```

Comment:
instead of creating new files in the repo, can you instead create it on the fly ?

Example
https://github.com/uutils/coreutils/blob/ae87843487230a014cbe799807aec55fd9faa788/tests/by-util/test_ln.rs#L629-L635=

Reply:
Ah yeah okay, will do.

Reply:
This is done now.

Reply:
Oops, forgot to delete the files. Sorry, 1 more commit.